editor/src/components/navigator/navigator-item/component-picker.tsx

```diff
@@ -176,6 +177,7 @@ const FilterBar = React.memo((props: FilterBarProps) => {
       onChange={handleFilterChange}
       value={filter}
       data-testId={componentPickerFilterInputTestId}
+      {...dataPasteHandler(true)}
```

Comment:
Is there a reason to use a helper in one of the cases, and just put the prop there in the other case?

Reply:
because this is not a `StringInput` but a regular `input` :)

Reply:
tbh I'm not sure it's worth using this helper rather than just inlining `[DataPasteHandler]: true` each time, since at the very least this is going to be recreating and spreading an object, which I _think_ would be cause issues with any `React.memo` components 🤔

Reply:
just a thought - maybe we can have higher level components (such as StringInput) just pass all the `data-` props down (and use only `data-pasteHandler`?

Reply:
tbh if we prefer not to use the spread (I did not know it could cause issues with memoization!) we can/should just use `data-pastehandler` directly :/

Reply:
@Rheeseyb I think that React.memo checks equality of the props themselves, regardless of how they were created

Reply:
Oh sorry of course it won't mess with the memoization!